src/UICrafter.Core/Components/Snackbar/ApiResponseStatusSnackbar.razor

```diff
@@ -0,0 +1,38 @@
+@inject ISnackbar Snackbar
+
+
+
+@code {
+
+    [Parameter]
+    public required string ResponseCode { get; set; }
+
+    protected override void OnInitialized()
+    {
+        switch (ResponseCode)
```

Comment:
Same thing as previous comment. Maybe extract it to a extension method if you will be using it a lot.